src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -118,6 +118,18 @@ class DriftTable extends Component {
         return row;
     }
 
+    renderSortButton(props) {
+        if (props.sort === 'none') {
+            return <SortIcon onClick={ () => this.props.toggleFactSort('asc') }/>;
+        }
+        else if (props.sort === 'asc') {
+            return <AngleUpIcon onClick={ () => this.props.toggleFactSort('desc') }/>;
+        }
+        else if (props.sort === 'desc') {
+            return <AngleDownIcon onClick={ () => this.props.toggleFactSort('asc') }/>;
+        }
```

Comment:
I think it can be a bit cleaner to set these equal to a variable and do a single return at the end.